src/checklist-ui/src/App.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
```

Comment:
I think this test will fail and will cause issues when we add the build pipeline